packages/imagemagick7.rb

```diff
@@ -51,6 +45,7 @@ def self.patch
   end
 
   def self.build
+    FileUtils.ln_s "#{CREW_PREFIX}/include/locale.h", "#{CREW_PREFIX}/include/xlocale.h" 
```

Comment:
This should be in the `def self.install` section also so it is tracked.  See below:
```
FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/include"
FileUtils.ln_s "#{CREW_PREFIX}/include/locale.h", "#{CREW_DEST_PREFIX}/include/xlocale.h"
```

Reply:
It has to be defined before the build step otherwise it  won't compile. So : shall I correct this part and do the same in the install step ?

Reply:
I'm just saying if you don't include in the `def self.install` section also, uninstalling this package won't remove that symlink.

Reply:
Ok. I'll add this in both places so with a try catch to avoid `File exists` error

Reply:
It looks like locale.h is in the glibc package so it probably makes more sense to include the xlocale.h symlink there instead.  That said, it is a core package that would require more extensive testing.

Reply:
It would indeed be better to do so. I put this ln -s here to avoid this trouble but, if you guide me (which tests are required : packages relying on glibc ?), I will handle this since it would be a cleaner approach

Reply:
It's not a nimble task because almost everything depends on glibc.  The most important thing is to not break install.sh.

Reply:
True, but since it is just an additional file used by nobody else but imagemagick7, files produced during glibc compilation won't be modified and then packages built upon glibc won't be affected since we don't upgrade glibc. No ?

Reply:
In theory, yes.  In practice, maybe?  Still need to test.